### PR TITLE
Updated Changes as preparation for release

### PR DESCRIPTION
**Pull request by matsduf**

### CODE REVIEW

File: Changes [hunk 1]
```diff
@@ -1,5 +1,23 @@
 Release history of Zonemaster
 
+v2018.1 2018-06-25 Public release
+
+High lights:
+
+- Zonemaster-LDNS is now built on latest version if LDNS and has been
```
Comment (sandoche2k) [inline Changes]: Zonemaster-LDNS is now built on LDNS latest version and has been

File: Changes [hunk 2]
```diff
@@ -1,5 +1,23 @@
...
+  modulized to make it possible to use available LDNS library.
+- Zonemaster-GUI has been rewritten from ground up.
+
+This version of Zonemaster also consists of the following components. For each
+component, see its Changes file.
+
+  Zonemaster-LDNS     v1.1.0   https://github.com/dotse/zonemaster-ldns
```
Comment (sandoche2k) [inline Changes]: https://github.com/dotse/zonemaster-ldns/blob/develop/Changes

Comment (sandoche2k) [inline Changes]: https://github.com/dotse/zonemaster-ldns/blob/develop/Changes

File: Changes [hunk 3]
```diff
@@ -1,5 +1,23 @@
...
+  Zonemaster-Engine   v2.0.7   https://github.com/dotse/zonemaster-engine
```
Comment (sandoche2k) [inline Changes]: https://github.com/dotse/zonemaster-ldns/blob/master/Changes

File: Changes [hunk 4]
```diff
@@ -1,5 +1,23 @@
...
+  Zonemaster-CLI      v1.1.3   https://github.com/dotse/zonemaster-cli
```
Comment (sandoche2k) [inline Changes]: https://github.com/dotse/zonemaster-cli/blob/master/Changes

File: Changes [hunk 5]
```diff
@@ -1,5 +1,23 @@
...
+  Zonemaster-Backend  v2.1.0   https://github.com/dotse/zonemaster-backend
```
Comment (sandoche2k) [inline Changes]: https://github.com/dotse/zonemaster-backend/blob/master/Changes

File: Changes [hunk 6]
```diff
@@ -1,5 +1,23 @@
...
+  Zonemaster-GUI      v2.0.0   https://github.com/zonemaster/zonemaster-gui
```
Comment (sandoche2k) [inline Changes]: https://github.com/zonemaster/zonemaster-gui/blob/master/Changes

Note : Add the changes file in GUI. This note is not to be included

File: Changes (hunk 2 above)
Comment (matsduf) [reply Changes]: Then it should be the master branch, not the develop branch. But I think it makes more sense to point at the entire component when we say "This version of Zonemaster also consists of the following components".

File: Changes (hunk 6 above)
Comment (matsduf) [reply Changes]: Do you want me to create the Changes file?

File: Changes (hunk 2 above)
Comment (sandoche2k) [reply Changes]: Yes, I thought i made a edit. It is below.

File: Changes (hunk 6 above)
Comment (sandoche2k) [reply Changes]: If you can.. otherwise i will discuss with Pierre and create one.

File: Changes (hunk 2 above)
Comment (matsduf) [reply Changes]: I still think it makes more sense to point at the component. In the release tag in Github I will have links to component, release tag and Changes. E.g. see https://github.com/dotse/zonemaster/releases/tag/v2017.4.2

**Pull request merged**